internal/db/models.go

```diff
@@ -122,8 +122,7 @@ type ProjectResource struct {
 	Name                     liquid.ResourceName `db:"name"`
 	Quota                    Option[uint64]      `db:"quota"`
 	BackendQuota             Option[int64]       `db:"backend_quota"`
-	MinQuotaFromBackend      Option[uint64]      `db:"min_quota_from_backend"`
-	MaxQuotaFromBackend      Option[uint64]      `db:"max_quota_from_backend"`
+	Forbidden                Option[bool]        `db:"forbidden"`
```

Comment:
This does not need to be an Option. LIQUID does not allow None here.
```suggestion
	Forbidden                bool        `db:"forbidden"`
```

And then consequently, all other places looking at Forbidden should also not use Option.